examples/features/interceptor/README.md

```diff
@@ -17,102 +17,104 @@ go run client/main.go
-stream operation interception.
-
-For pre-processing, it's similar to unary interceptor.
-
-However, rather than doing the RPC method invocation and post-processing
-afterwards, stream interceptor intercepts the users' operation on the stream.
-First, the interceptor calls the passed-in `streamer` to get a `ClientStream`,
-and then wraps around the `ClientStream` and overloading its methods with
-intercepting logic. Finally, interceptors returns the wrapped `ClientStream` to
-user to operate on.
-
-In the example, we define a new struct `wrappedStream`, which is embedded with a
-`ClientStream`. Then, we implement (overload) the `SendMsg` and `RecvMsg`
-methods on `wrappedStream` to intercept these two operations on the embedded
+The type for client-side stream interceptors is
+[`StreamClientInterceptor`](https://godoc.org/google.golang.org/grpc#StreamClientInterceptor).
+It is a function type with signature: `func(ctx context.Context, desc
+*StreamDesc, cc *ClientConn, method string, streamer Streamer, opts
+...CallOption) (ClientStream, error)`. An implementation of a stream interceptor
+usually includes pre-processing, and stream operation interception.
+
+The pre-processing is similar to unary interceptors.
+
+However, rather than invoking the RPC method followed by post-processing, stream
+interceptors intercept the users' operations on the stream. The interceptor
+first calls the passed-in `streamer` to get a `ClientStream`, and then wraps
+around the `ClientStream` while overloading its methods with the interception
+logic. Finally, the interceptor returns the wrapped `ClientStream` to user to
+operate on.
+
+In the example, we define a new struct `wrappedStream`, which embeds a
+`ClientStream`. We then implement (overload) the `SendMsg` and `RecvMsg` methods
+on `wrappedStream` to intercept these two operations on the embedded
 `ClientStream`. In the example, we log the message type info and time info for
 interception purpose.
 
-To install the stream interceptor for a ClientConn, configure `Dial` with
-`DialOption`
-[`WithStreamInterceptor`](https://godoc.org/google.golang.org/grpc#WithStreamInterceptor).
+To install a stream interceptor for a ClientConn, configure `Dial` with the
+[`WithStreamInterceptor`](https://godoc.org/google.golang.org/grpc#WithStreamInterceptor)
+`DialOption`.
 
 ### Server-side
 
-Server side interceptor is similar to client side, though with slightly
-different provided info.
+Server side interceptors are similar to client side interceptors, with slightly
+different information provided as args.
 
 #### Unary Interceptor
 
-[`UnaryServerInterceptor`](https://godoc.org/google.golang.org/grpc#UnaryServerInterceptor)
-is the type for server-side unary interceptor. It is a function type with
-signature: `func(ctx context.Context, req interface{}, info *UnaryServerInfo,
-handler UnaryHandler) (resp interface{}, err error)`.
+The type for server-side unary interceptor is
```

Comment:
"for _a_ s-s unary" or "for s-s- unary interceptor_S_" please (you use the latter below so best to stay consistent).

Reply:
Changed to "server-side unary interceptors".